app/views/projects/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<%-# TODO: tables should be variable width depending on how many years of data we have -%>
```

Comment:
I actually suggest putting something like `TODO(isaiahyoon): tables should ...`. (That means that the next person who comes along will know to ask you about what you meant not necessarily that you'll do it.)